Release/CHECKLIST

```diff
@@ -19,5 +19,6 @@
     string "__PREFIX__" rather than your local prefix
 [x] Remove __collect_safe from generated chez (to avoid need for chez >9.5)
 [x] Tag on github with version number (in the form vX.Y.Z)
+[ ] Add "tppc32osx tppc64osx" into bootstrap/idris2_app/idris2.ss
```

Comment:
* Could you please add two sub-checkboxes to this? One specifying we need to append the architectures to the list before `load-shared-object "libc.dylib")`, and one specifying that this also needs to be done in the `[...]Chez-schHeader` and `[...]ChezSep-schHeader` functions.
* Nit: I would reword to "Patch support for 'tppc[...]' into [...]"

Reply:
Done